app/src/main/java/nl/nfi/cellscanner/recorder/LocationRecordingService.java

```diff
@@ -0,0 +1,140 @@
+package nl.nfi.cellscanner.recorder;
+
+import android.annotation.SuppressLint;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+import android.support.v4.app.NotificationCompat;
+import android.telephony.CellInfo;
+import android.telephony.TelephonyManager;
+import android.util.Log;
+
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import nl.nfi.cellscanner.App;
+import nl.nfi.cellscanner.Database;
+import nl.nfi.cellscanner.MainActivity;
+import nl.nfi.cellscanner.R;
+
+public class LocationRecordingService extends Service {
+
+    private static final int NOTIF_ID = 123;
```

Comment:
Willen we hier nog een TODO voor een ander nummer, of gaan we er vanuit dat 123 altijd beschikbaar is?